aperturedb/Connector.py

```diff
@@ -564,10 +564,10 @@ def last_query_ok(self):
         return self.check_status(self.response) >= 0
 
     def check_status(self, json_res):
-
-        status = 0
-        if (isinstance(self.response, dict)):
-            if ("status" not in self.response):
+        # Default status is -2, which is an error, but not a server error.
+        status = -2
+        if (isinstance(json_res, dict)):
+            if ("status" not in json_res):
```

Comment:
We should have a definition of statuses somewhere. -1 is server error, -2 is local, 0 is ok and 1+ are server responses?

Reply:
updating the doc string and types.